### PR TITLE
Select Condition component

### LINKED ISSUE #73: [FE] - Split Position - Select Condition
- Columns to show - TBD
- Add button in Split Position
- Create new modal
- Table (similar to condition list)
- Search and filter
- Select condition on click
- Submit

### PR DESCRIPTION
**Pull request by lmcorbalan**
- Add condition id imput + select condition modal
- Uses new component in Split Position and Merge Positions

closes #73

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr129--conditionaltokensexplorer.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr129--conditionaltokensexplorer.review.gnosisdev.com

File: src/components/form/SelectCondition/index.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+export const SelectCondition = ({ onChange }: { onChange?: (id: string) => void }) => {
+  const { condition, errors, loading, setConditionId } = useConditionContext()
+  const [conditionId, setManualConditionId] = React.useState<string>('')
+  const [isModalOpen, setIsModalOpen] = React.useState(false)
+
+  const debouncedHandler = useDebounceCallback((id) => {
+    setConditionId(id)
+  }, 500)
+  const imputHandler = useCallback(
```
Comment (mariano-aguero) [inline src/components/form/SelectCondition/index.tsx:34]: Please, if you use ̣`React.useState` as in line 29, use in this line `React.useCallback` to keep consistency, also in the lines 43 and 48, and remove the unused import

File: src/components/form/SelectCondition/index.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    <>
+      <Row cols={'1fr'} marginBottomXL>
+        <TitleValue
+          title="Condition Id"
+          titleControl={<TitleControl onClick={openModal}>Select Condition</TitleControl>}
+          value={
+            <>
+              <Textfield
+                error={!!errors.length}
+                // name="conditionId"
```
Comment (mariano-aguero) [inline src/components/form/SelectCondition/index.tsx:64]: Remove this if is not necessary

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr129--conditionaltokensexplorer.review.gnosisdev.com

**Pull request merged**